app/helpers/i18n_helper.rb

```diff
@@ -0,0 +1,17 @@
+module I18nHelper
+
+  private
```

Comment:
`private` has no effect in modules and besides I don't really understand what you are trying to achive with it.